golang.req

```diff
@@ -21,6 +33,9 @@ end
 requires = []
 
 unless filelist.empty? then
+
+	requires << "golang(API) = " + go_get_version()
```

Comment:
@jordimassaguerpla It's easy to distinguish the 2 cases...executables locate in $GOBIN. it's easy to detect if there's any binary. but I think people from SUSE were talking about some fundamental changes: from shipping `.a` to `.go`. I call it fundamental because golang-packaging actually unpacks those `.a` to find Provides/Requires.